typescript/audio-summarizer/with-literal/src/app/api/transcribe/route.ts

```diff
@@ -0,0 +1,87 @@
+import { Attachment, LiteralClient } from "@literalai/client";
+import { NextRequest, NextResponse } from "next/server";
+import OpenAI, { toFile } from "openai";
+import { ReadableStream } from "stream/web";
+import { Readable } from "stream";
+
+const openai = new OpenAI();
+const literalClient = new LiteralClient();
+
+const threadName = "Emojifier Run Thread";
+
+export async function POST(req: NextRequest) {
+  const data = await req.formData();
+  const formAudio = data.get("audio") as File;
+  const runId = data.get("runId") as string;
+
+  if (!formAudio) {
+    return new NextResponse("No audio file provided", { status: 400 });
+  }
+
+  // Create or retrieve the thread
+  const thread = await literalClient.thread({ name: threadName }).upsert();
```

Comment:
I'd mention in comments that, for production usage, failure to run actions against Literal AI shouldn't stop the app from working (with try/catch).

Reply:
that's a great point, and one that seems to apply to most of our cookbooks :thinking: i could add it as a caveat in the README, wdyt ?

Reply:
Great idea!

Reply:
come to think of it there may be a better place for this (important) information. Maybe the TS SDK ?